site2/docs/getting-started-standalone.md

```diff
@@ -23,6 +23,40 @@ Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**.
 > **Note**  
 > Broker is only supported on 64-bit JVM.
 
+#### Installing JDK on M1
```

Comment:
We're going to fix this for 2.11 (upgrading RocksDB) so we should take a note to remove this portion of docs once that release is out.

Reply:
Where shall I place this note @merlimat ? An issue?

Reply:
Ok. I moved the changes to 2.10.0